CONTRIBUTING.md

```diff
@@ -160,7 +160,7 @@ development tools:
 We currently have several monitoring tools in place:
 
 - New Relic, mainly for basic pings
-- A custom script powered by Mandrill to e-mail us nightly status reports in
+- A custom script powered by slacks us nightly status reports about
```

Comment:
Very nice to see the whitespace cleanup in here!

For this particular line, it looks like we're missing a word or two.  What about this to make it easier to understand?

`A custom script powered by Slack that posts a nightly status update about...`